### PR TITLE
Update grid gutters to 24px on code.org

### PR DESCRIPTION
**Pull request by kelbyhawn**
Update the gutters between common grid and flexbox elements to `24px`/`1.5rem`.

_Note:_ I didn't update any gutters on the carousels since I'll be refactoring those shortly.

**Jira ticket:** https://codedotorg.atlassian.net/browse/ACQ-1056

----

## Two-col action blocks
| Before | After |
| ----- | ----- |
| Block-2-col-before | Block-2-col-after |

## Three-col action blocks
| Before | After |
| ----- | ----- |
| Block-3-col-before | Block-3-col-after |

## `.video-series` container
| Before | After |
| ----- | ----- |
| Video-before | Video-after |

## `.button-wrapper` container
| Before | After |
| ----- | ----- |
| Buttons-before | Buttons-after |

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss [hunk 1]
```diff
@@ -1073,6 +1074,10 @@ section.quote {
     margin: 0 !important;
     width: 100%;
   }
+
+  .video_caption_link {
+    margin-bottom: 0;
+  }
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss:1084]: This is cleaning up extra vertical space under video captions.

File: pegasus/sites.v3/code.org/public/teach.haml [hunk 2]
```diff
@@ -85,7 +85,7 @@ theme: responsive_full_width
     %hr.dark
     %div.centered
       %h4=hoc_s(:teach_page_free_curriculum_heading_02)
-      %div.flex-container.justify-center.wrap{style: "gap: 1rem"}
+      .button-wrapper{style: "justify-content: center"}
```
Comment (TurnerRiley) [inline pegasus/sites.v3/code.org/public/teach.haml]: Could this use the change you just made [here](https://github.com/code-dot-org/code-dot-org/pull/55637)?

**Review by TurnerRiley — APPROVED** [review]
One quick question, but otherwise looks great!

File: pegasus/sites.v3/code.org/public/teach.haml (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/code.org/public/teach.haml]: Yep! I'm expecting to get a merge conflict, and will make sure it has that new version 😄

**Pull request merged**